<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>

  <!-- Google tag (gtag.js) -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-47BLR7P19B">
</script>
<script>
  window.dataLayer = window.dataLayer || []; function
  gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config',
  'G-47BLR7P19B');
</script>

<link rel="stylesheet" href="css/screen.css" type="text/css" media="screen, projection" />
<link rel="stylesheet" href="css/print.css" type="text/css" media="print" />
<!--[if IE]>
  <link rel="stylesheet" href="../css/ie.css" type="text/css" media="screen, projection">
<![endif]-->
<link rel="stylesheet" href="css/style.css" type="text/css" media="screen, projection" />
<title>Matthew Berginski's Personal Webpage</title>
</head>

<body>
    <div class="container">
        <h1>Matthew Berginski's Personal Webpage</h1>

        <div class="prefix-1">
        	<IMG src="images/self_picture.jpg" width=250 alt="Picture of Matthew Berginski">
	</div>
        
	<h2>Contact Information</h2>
	
	<div class="prefix-1">
		   
            <div class="span-4 clear">
	        <strong>Phone</strong>
	    </div>
	    <div class="span-18">
	        859-797-6722
	    </div>
            <br />
		
		    <div class="span-4 clear">
		        <strong>Email</strong>
		    </div>
		    <div class="span-18">
		        matthew.berginski [AT] gmail.com OR matthew.berginski [AT] tempus.com
		    </div>
            <br />
            <br />
	</div>
        
        <div class="span-24">
			<br />
        <hr>
        <p />
        
        I am a computational biologist at <a href="https://www.tempus.com/">Tempus Labs</a>, where I specialize in using machine learning and other computational techniques to analyze, understand and use biological data.
        
        <p />
	Previously, I was a Research Scientist in the Pharmacology department at the
	University of North Carolina and I worked in Shawn Gomez's <a href =
	"http://gomezlab.bme.unc.edu/">lab</a> and Gary Johnson's <a
	href="http://www.med.unc.edu/pharm/johnsonlab/">lab</a>. I specialized
	in bioinformatics with a focus on image analysis, machine learning and quantification of
	high-throughput data. I also worked with the 
	<a href="https://www.med.unc.edu/barc/">Bioinformatics and Analytics Research Collaborative</a> (BARC) a bioinformatics core facility at UNC.  

    <p />
    I do most of my work using R, but have also written code using MATLAB, Perl and Python. You can find a listing of my public projects, publications and open source code here:

	<hr>
	<h3>Public Projects</h3>
	<ul>
		<li>
			<a href="https://faas.bme.unc.edu/">The Focal Adhesion Analysis Server</a>: A web-based image processing tool dedicated to processing images of focal adhesions. Focal adhesions are a sub-cellular structure that allow individual cells to connect to the environment. This tool has been used to process over 2,000,000 images. Source code available through <a href="https://github.com/mbergins/focal_adhesions">github</a>.
		</li>
		<li>
			<a href="https://darkkinome.org/">The Dark Kinome Knowledgebase</a>: A website for the disemination of results related to understudied kinases from an NIH-funded consortium. Source code available through <a href="https://github.com/IDG-Kinase/darkkinasekb">github</a>.
			<ul>
				<li><a href="http://expression.darkkinome.org/">The Dark Kinase Expression Browser</a>: A data browser dedicated understanding understudied kinase expression through data from the GTex and HPM consortiums. The application was developed using the R shiny framework. Source code is available through <a href="https://github.com/IDG-Kinase/kinase_expression">github</a>.
				</li>
			</ul>
		</li>
	</ul>
	<ul>
		<li>Shiny Applications: websites based around the R shiny framework that I've been part of building.</li>
		<ul>
			<li><a href="http://phanstiel-lab.med.unc.edu/CORAL/">CORAL</a>: Visualization application developed in Phanstiel lab for showing kinase-centric data in several forms (primary helped with deployment and bug testing). <a href="https://github.com/dphansti/CORAL">Github</a></li>
			<li><a href="http://parnell-lab.med.unc.edu/Embryo-Transcriptomics/">Embryonic Expression Transcriptomics</a>: Transcriptomics data browser developed in collaboration with Parnell Lab. <a href="https://github.com/mbergins/Embryo-Genes">Github</a></li>
			<li><a href="http://vogtviruslab.med.unc.edu/evd68_humanspinalcord/">EVD68 Spatial Transcripts</a>: Visualization tool for a spatial transcriptomics data set developed with the Vogt Lab. <a href="https://github.com/mbergins/evd68_humanspinalcord_spatial_transcripts">Github</a></li>
			</li>
		</ul>
	</ul>

	<h3>Preview Applications</h3>
	<ul>
		<li>
			<a href="http://kipred.gomezlab.org">Kinotype Viability Predictor</a>: A shiny framework based tool that accepts RNAseq results and then predicts the effect on cell viability of ~230 kinase inhibitors. <a href="https://github.com/gomezlab/kinase_inhibitor_pred_app">github</a>
		</li>
	</ul>

	<p />
	Source code for the other projects I've worked on are available through <a href="https://github.com/mbergins">github</a>.
	
	<p />
	A list of my publications is available through <a href="https://scholar.google.com/citations?user=-n9FUI0AAAAJ&hl=en">google scholar</a>.
        </div>
        <br>
        <hr>
        <p>
        <a href="documents/Berginski_CV.pdf">My CV</a>
        </p>
        <br />
        
    </div>
</body>
</html>
